packages/fx-core/src/plugins/solution/fx-solution/arm.ts

```diff
@@ -325,43 +320,30 @@ export async function getParameterJson(ctx: SolutionContext) {
     throw new Error("Failed to get target environment name from solution context.");
   }
 
-  let parameterFileName, parameterFolderPath, parameterTemplateFilePath;
-  if (isMultiEnvEnabled()) {
-    parameterFileName = parameterFileNameTemplateNew.replace("@envName", ctx.envInfo.envName);
-    parameterFolderPath = path.join(ctx.root, configsFolder);
-    parameterTemplateFilePath = path.join(
-      path.join(ctx.root, templateFolderNew),
-      parameterTemplateFileName
-    );
-  } else {
-    parameterFileName = parameterFileNameTemplate.replace("@envName", ctx.envInfo.envName);
-    parameterFolderPath = path.join(ctx.root, baseFolder, parameterFolder);
-    parameterTemplateFilePath = path.join(parameterFolderPath, parameterTemplateFileName);
-  }
+  const parameterFileName = parameterFileNameTemplate.replace("@envName", ctx.envInfo.envName);
+  const parameterFolderPath = path.join(ctx.root, configsFolder);
+  const parameterEnvFilePath = path.join(
+    path.join(ctx.root, configsFolder),
+    parameterFileDefaultTemplate
+  );
 
   const parameterFilePath = path.join(parameterFolderPath, parameterFileName);
   let createNewParameterFile = false;
   try {
     await fs.stat(parameterFilePath);
   } catch (err) {
     ctx.logProvider?.info(
-      `[${PluginDisplayName.Solution}] ${parameterFilePath} does not exist. Generate it using ${parameterTemplateFilePath}.`
+      `[${PluginDisplayName.Solution}] ${parameterFilePath} does not exist. Generate it using ${parameterEnvFilePath}.`
```

Comment:
Since we do not have a parameter template now, missing parameter file for an environment should report error instead of copy the file from "dev" env. Please help update related logic.

Reply:
sure